Add blog post: Introducing the nf-core newsletter (#4272)
nf-core/website: sites/main-site/src/content/blog/2026/newsletter-launch.mdx

In a normal month there are a handful of pipelines with new releases, one or two typically have their very first release, there are new blog posts, bytesize talks, training events, hackathons and all sorts.
If you don't follow Slack and the website every day, it's easy to lose track of all of it.

To try to help with this, we built a newsletter.
Once a month it gathers up what happened across nf-core onto a single page, and then emails that to you.
You can sign up below:

<NewsletterSignup compact={true} />

We'll also publish the newsletter [on LinkedIn](https://www.linkedin.com/newsletters/nf-core-7474718981564706816/), so you can subscribe there instead if you prefer.

The whole thing is rendered on the website, so you can read the latest edition and browse previous months at [nf-co.re/newsletter](/newsletter).

## What's in an edition

Each month the newsletter pulls together:

- New blog posts
- Security advisories
- Pipeline releases (with brand-new pipelines' first releases highlighted)
- Upcoming bytesize talks, trainings, and hackathons
- Newly minted pipeline repos, plus pipeline and RFC proposals from [nf-core/proposals](https://github.com/nf-core/proposals/issues)

> There's also an "in case you missed it" section that shows blogs and other things from the past couple of months, in case something slipped past you the first time.

Reviewer: ```suggestion
There's also an "in case you missed it" section that shows blog posts and other things from the past couple of months, in case something slipped past you the first time.
```

---
title: Introducing the nf-core newsletter
subtitle: "Community news, pipeline releases and events, gathered once a month and emailed to you if you want it"
pubDate: 2026-06-23T09:00:00+01:00
headerImage: /assets/images/blog/newsletter-launch/newsletter-banner.jpg
headerImageAlt: A blue rounded button with a white envelope icon, representing email

> headerImageDim: [2400, 1350]

Reviewer: ```suggestion
```
not needed if you don't want to make it a lot smaller then the actual image

authors:
    - ewels
label:
    - community
---

import NewsletterSignup from "@components/newsletter/NewsletterSignup.astro";

# Introducing the nf-core newsletter

The nf-core community moves fast.
In a normal month there are a handful of pipelines with new releases, one or two typically ship their very first release, there are new blog posts, bytesize talks, training events, hackathons and all sorts.
If you don't follow Slack and the website every day, it's easy to lose track of all of it.

So, we built a newsletter. Once a month it gathers what happened across nf-core onto a single page, and you can have that page emailed to you.
Sign up below:

<NewsletterSignup compact={true} />

> You can also sign up to read it [on LinkedIn](https://www.linkedin.com/newsletters/nf-core-7474718981564706816/).

Reviewer: ```suggestion
```
or mention rss as well. but feels like duplicated information with respect to the section later

Author: I don't mind duplication - getting people to sign up and follow is the main intention of the blog post, so a little repetition on that front (bracketing the content) was intentional.

Author: Tweaked in 6db7586
